src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -16,14 +16,11 @@
 (def animation-delay 450)
 
 (defn with-animation
-  [value & [options callback]]
-  (reanimated/with-spring
-   value
-   (clj->js (merge {:mass      2
-                    :stiffness 500
-                    :damping   200})
-            options)
-   callback))
+  [value]
+  (reanimated/with-spring value
```

Comment:
FYI: The newer ClojureScript version throws a runtime error when the original function signature is used. I just removed the extra arguments, since they were never used in the first place.